src/Stripe.net/Infrastructure/Middleware/ParserPlugin/DateFilterPlugin.cs

```diff
@@ -14,7 +14,7 @@ public bool Parse(ref string requestString, JsonPropertyAttribute attribute, Pro
                 return false;
             }
 
-            var filter = (StripeDateFilter) propertyValue;
+            var filter = (StripeDateFilter)propertyValue;
```

Comment:
While I implemented that one, I feel quite strongly that this missing space is not nice to look at and reason with.